app/controllers/administrate/application_controller.rb

```diff
@@ -117,11 +123,14 @@ def resource_params
         permit(dashboard.permitted_attributes)
     end
 
-    delegate :resource_class, :resource_name, :namespace, :resource_scope,
-             to: :resource_resolver
+    delegate :resource_class, :resource_name, :namespace, to: :resource_resolver
     helper_method :namespace
     helper_method :resource_name
 
+    def dashboard_class
```

Comment:
Yes I think so.  If I'm honest I'm drawing a blank as to why I even changed the "dashboard" function at all.